Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -598,6 +600,12 @@ extension TelemetryWrapper {
         case (.action, .tap, .requestMobileSite, _, _):
             GleanMetrics.PageActionMenu.requestMobileSite.add()
 
+        // Firefox Homepage
+        case (.action, .tap, .firefoxHomepage, EventValue.yourLibrarySection.rawValue, let extras):
```

Comment:
And here?

Reply:
On line 446 we have `let value = value?.rawValue ?? ""`. This means either I use the rawValue, or I have to refactor everything else. I'd prefer to do it the clunky rawValue way, and save a proper refactor for later where we can make things make sense. haha